src/index.ts

```diff
@@ -517,6 +534,16 @@ export class Bigtable {
     this.appProfileId = options.appProfileId;
     this.projectName = `projects/${this.projectId}`;
     this.shouldReplaceProjectIdToken = this.projectId === '{{projectId}}';
+
+    if (options.collectMetrics === false) {
+      this.collectMetrics = false;
+      this.metricsHandlers = [];
+    } else {
+      this.collectMetrics = true;
+      this.metricsHandlers = [
+        new GCPMetricsHandler(new CloudMonitoringExporter()),
```

Comment:
is there a reason to have `CloudMonitoringExporter` as an argument? Will a different one ever be passed in?

Reply:
Within the client library code `CloudMonitoringExporter` is always used, but tests sometime pass in a mock exporter so that we can unit test GCPMetricsHandler's behaviour. If you want we can define a DefaultGCPMetricsHandler class that just always uses the CloudMonitoringExporter to eliminate the confusion.

Reply:
It looks like [we discussed this in the other PR](https://github.com/googleapis/nodejs-bigtable/pull/1583#discussion_r2034178097). But this one probably makes more sense

Isn't there a way to inject the mock without making the internal variable part of the API surface?

Reply:
In multiple tests a mock exporter is passed in here:

![image](https://github.com/user-attachments/assets/6d928ae0-48c1-4ee1-a947-706dc0785d1f)

We could inject the mock with proxyquire, doing something like https://github.com/googleapis/nodejs-bigtable/pull/1571/commits/1ddaa7df1d142abff4fe399a2da6fa8b1cf00aaf for each test, but when proxyquire is involved it can be hard to debug later. What do you want to do here?